app/helpers/template_helpers.py

```diff
@@ -94,6 +94,14 @@ def service_links_context(
 
         return None
 
+    @property
+    def data_layer_context(
+        self,
+    ) -> Union[list[dict], None, list]:
```

Comment:
We should always return a list that may contain dicts since we just pass this directly to the view.

Reply:
As above, fixed now, thanks!